style.css

```diff
@@ -0,0 +1,181 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap");
+
+:root {
+  --color-white: #f3f3f3;
+  --color-darkblue: #1b1b32;
+  --color-darkblue-alpha: rgba(27, 27, 50, 0.8);
+  --color-green: #37af65;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: var(--color-white);
+  margin: 0;
+}
+
+/* mobile friendly alternative to using background-attachment: fixed */
```

Comment:
- [ ] Please as I request, it would be better if you replace this comment with another one explaining another issue like linter errors or stying problems because this comment in the code is explanatory and it doesn't affect the cleanliness of the code
![image](https://user-images.githubusercontent.com/74744787/160457238-94222e1b-6c49-498d-a9f9-ec94b3fb3687.png)

Reply:
sorry, I did not get your point.

Reply:
> sorry, I did not get your point.

@sameh080081 in the review you said that the programmer shouldn't comment at all but this is not true because it is a good thing to make explanatory comments although it is not a good thing to write a code and comment on it and keep it. After all, it will make your code look missy I tried to reach you on slack but I couldn't know which one was your account because there is no profile picture.